app/src/renderer/vuex/modules/wallet.js

```diff
@@ -9,7 +9,8 @@ export default ({ commit, node }) => {
     key: { address: "" },
     history: [],
     historyLoading: false,
-    denoms: []
+    denoms: [],
+    zoneIds: ["cosmos-hub-1", "cosmos-hub-2"]
```

Comment:
Should we maybe put this in the config file? Can be done later.

Reply:
Yes, for sure. I wasn't clear on how the backend would be implemented so I left it in `wallet.js`